src/arraymancer/laser/tensor/datatypes.nim

```diff
@@ -177,6 +177,17 @@ func unsafe_raw_offset*[T: KnownSupportsCopyMem](t: var Tensor[T], aligned: stat
   ## and that the data is aligned by LASER_MEM_ALIGN (default 64).
   unsafe_raw_offset_impl(t.offset)
 
+func unsafe_raw_data*[T: KnownSupportsCopyMem](t: Tensor[T], aligned: static bool = true): UncheckedArray[T] {.inline.} =
```

Comment:
Will comment more later, but this should be `ptr UncheckedArray[T]` as the return value, surely?

Reply:
I can't believe I've overlooked that